dbms/src/Flash/Coprocessor/DAGUtils.cpp

```diff
@@ -44,6 +44,14 @@ extern const String uniq_raw_res_name;
 
 namespace
 {
+const std::unordered_map<tipb::ExprType, String> window_func_map({
+    {tipb::ExprType::Rank, "rank"},
+    {tipb::ExprType::DenseRank, "dense_rank"},
+    {tipb::ExprType::RowNumber, "row_number"},
+    {tipb::ExprType::Lag, "lag_in_frame"},
+    {tipb::ExprType::Lead, "lead_in_frame"},
```

Comment:
Looks like `lag_in_frame` and `lead_in_frame` is not supported yet?

Reply:
> Looks like `lag_in_frame` and `lead_in_frame` is not supported yet?

deleted.